### DISCUSSION TITLE
Impact of Dart workspace introduced in 3.6

### DISCUSSION THREAD

**Discussion by v41e** [invertase/melos#802] [Q&A] [answered]
Hi Melos team 👋,

Dart recently added native support for workspaces, at first glance it seems pretty useful out of the box, but some essential features that Melos provides are missing. I hope Melos will still exist and take advantage of the new workspace feature and not be discontinued.

**Comment by spydon** [comment, marked as answer]
Duplicate of the pinned issue: https://github.com/invertase/melos/issues/747
